### ISSUE TITLE
Error when using dirhash command

### ISSUE THREAD

**Issue by fschleich** [ascmitc/mhl#96]
I get the following error when attempting to create a directory hash:

```
fschleich@fschleichmacpro ~ % ascmhl dirhash /Users/fschleich/Testmaterial/TestPackages/Meridian_HD_SDR_TEST 
Traceback (most recent call last):
 File "/usr/local/bin/ascmhl", line 8, in <module>
  sys.exit(mhltool_cli())
 File "/Library/Python/3.8/site-packages/click/core.py", line 1137, in __call__
  return self.main(*args, **kwargs)
 File "/Library/Python/3.8/site-packages/click/core.py", line 1062, in main
  rv = self.invoke(ctx)
 File "/Library/Python/3.8/site-packages/click/core.py", line 1668, in invoke
  return _process_result(sub_ctx.command.invoke(sub_ctx))
 File "/Library/Python/3.8/site-packages/click/core.py", line 1404, in invoke
  return ctx.invoke(self.callback, **ctx.params)
 File "/Library/Python/3.8/site-packages/click/core.py", line 763, in invoke
  return __callback(*args, **kwargs)
 File "/Library/Python/3.8/site-packages/ascmhl/commands.py", line 931, in directory_hash
  dir_hash_context.append_hash(hash_string, item_name)
AttributeError: 'DirectoryHashContext' object has no attribute 'append_hash'
```

**Comment by ptrpfn** [comment]
Looks like that got lost in merging.. Will take a look.

**Comment by ptrpfn** [comment]
Comments in the source code reminded me that this is an old command that is deprecated (and also not currently documented). The open task was to move its use case into the `verify` command.

This issues is solved with #101.
